client/unit-tests/coretypes/test_filtering.py

```diff
@@ -58,15 +58,15 @@ def test__format_filter(geojson):
 
     filter_from_constraints = _format_filter(
         [
-            Dataset.name.in_(["a", "b", "c"]),
-            Model.name.in_(["x", "y", "z"]),
-            Label.score > 0.75,
-            Annotation.polygon.area > 1000,
-            Annotation.polygon.area < 5000,
-            Dataset.metadata["some_str"] == "foobar",
-            Dataset.metadata["some_float"] >= 0.123,
-            Dataset.metadata["some_datetime"] > datetime.timedelta(days=1),
-            Dataset.metadata["some_geospatial"].intersect(geojson),
+            Dataset.name.in_(["a", "b", "c"]),  # type: ignore - filter type error
```

Comment:
I generally ignored `Filter` statements since Charles is reworking filters at the moment

Reply:
> I generally ignored `Filter` statements since Charles is reworking filters at the moment. I marked these statements with `# type: ignore - filter type error` so we can ctrl + F for them later